Package bluetuith is broken at version 0.1.8

### Describe the bug
Package `bluetuith` is broken at version 0.1.8. It still worked at version 0.1.7, so I assume that https://github.com/NixOS/nixpkgs/pull/263322 is to blame.

I've verified that the issue is fixed in 0.1.9, so I assume all that's needed would be to kick the @r-ryantm bot. Not sure how to do that.

### Steps To Reproduce
```
>> bluetuith
panic: interface conversion: interface {} is nil, not bool

goroutine 1 [running]:
github.com/darkhz/bluetuith/bluez.(*Bluez).ConvertToDevices(0xc0001e2090?, {0xc00001f170, 0x25}, 0xc0001abbc0?)
        github.com/darkhz/bluetuith/bluez/device.go:193 +0x8cb
github.com/darkhz/bluetuith/bluez.(*Bluez).RefreshStore(0xc0001e2090)
        github.com/darkhz/bluetuith/bluez/bluez.go:89 +0x325
github.com/darkhz/bluetuith/bluez.NewBluez()
        github.com/darkhz/bluetuith/bluez/bluez.go:55 +0x85
main.main()
        github.com/darkhz/bluetuith/main.go:14 +0x1f
```

### Expected behavior
No panic

### Notify maintainers
@r-ryantm @ryantm @lovesegfault @theHedgehog0

### Metadata

```console
>> nix run nixpkgs#nix-info -- -m
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.0-1025-oem, Ubuntu, 22.04.3 LTS (Jammy Jellyfish), nobuild`
 - multi-user?: `no`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.1`
 - channels(root): `"nixpkgs"`
 - channels(mforster): `""`
 - nixpkgs: `not found`
```

--- a/pkgs/tools/bluetooth/bluetuith/default.nix
+++ b/pkgs/tools/bluetooth/bluetuith/default.nix
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "bluetuith";
-  version = "0.1.8";
+  version = "0.1.9";
 
   src = fetchFromGitHub {
     owner = "darkhz";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-b+J+8nxFZYJlAKOStpH7ItPqMw7inM5pss17kyX1brg=";
+    sha256 = "sha256-vdHnG0uQdy5PboIovtxl5i9xwFpjYLCZf2IGeiMcWe8=";
   };
 
-  vendorHash = "sha256-d0O54KNGLXU8FGr1eSEp30JMWNVo91Le2MI8UnAfTuU=";
+  vendorHash = "sha256-pYVEFKLPfstWWO6ypgv7ntAaE1Wmq2XKuZC2ccMa8Vc=";
 
   ldflags = [ "-s" "-w" ];
 

